### PR TITLE
Fix broken web link for SGX mitigation tools

### PR DESCRIPTION
**Pull request by manju956**
Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

**Review by rranjan3 — APPROVED** [review]

File: PREREQUISITES.md [hunk 1]
```diff
@@ -370,8 +370,8 @@ problems.
   mitigation applied on SGX CVEs as documented here [sgxsdk-guide](#https://github.com/intel/linux-sgx)
 
   ```bash
-  wget https://download.01.org/intel-sgx/latest/linux-latest/as.ld.objdump.gold.r2.tar.gz
-  tar -xvf as.ld.objdump.gold.r2.tar.gz
+  wget https://download.01.org/intel-sgx/latest/linux-latest/as.ld.objdump.gold.r3.tar.gz
```
Comment (manojgop) [inline PREREQUISITES.md:373]: Can we use the files relevant to the SGX release we use  for AValon. I assume we support 2.10. If so, please use files in https://download.01.org/intel-sgx/sgx-linux/2.10/
eg: https://download.01.org/intel-sgx/sgx-linux/2.10/as.ld.objdump.gold.r2.tar.gz

Comment (manojgop) [reply PREREQUISITES.md:373]: Another option is to use latest 2.12. In that case use files from

https://download.01.org/intel-sgx/sgx-linux/2.12/

Comment (manju956) [reply PREREQUISITES.md:373]: Earlier there was no sgx version specific mitigation tools available, hence we used latest. Now I do see, mitigation tools in https://download.01.org/intel-sgx/sgx-linux/2.10/ link. will try to use the same.

**Review by manojgop — APPROVED** [review]

**Comment by manju956** [comment]
This PR changes has been included in https://github.com/hyperledger/avalon/pull/719 and hence closing this PR

**Pull request closed without merging**